### ISSUE TITLE
How to add the parameter"ignore_index" when replace torch.nn.cross_entropy

### ISSUE THREAD

**Issue by JackJi12316** [agencyenterprise/uncertainty-loss#14]
Thanks for your work!
As title, I want to know how to change to achieve the parameter"ignore_index"
